mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -331,6 +332,23 @@ func ResourceContainerCluster() *schema.Resource {
 								},
 							},
 						},
+						"gcs_fuse_csi_driver_config": {
```

Comment:
For example, you'll want to move this block of code to line 423, and current line `5006` to `5056`. Does that make sense? And even references in the tests.

Reply:
Sorry for that.

I have updated them all in the latest iteration. Can you please check if it looks good.